N new.csv

```diff
@@ -0,0 +1,36 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+奶糖,/,noun,1886,,洪士提反譯『萬國藥方』卷一,
+難產,/,noun,1822,LABOUR（Difficult labour，拆副難產；難產. There is a volume on difficult labours, called 達生編 馬禮遜 英華字典,
+腦海,/,noun,1913,brains , Mateer Ada Haven Hand Book of New Terms and Newspaper Chinese,
+腦子,/,noun,unknown,腦子像石頭，總是說不通。,魯迅(1881-1936)『書信集‧致孟十還』, publication date absent in Schmidt
```

Comment:
```suggestion
腦子,/,noun,unknown,腦子像石頭，總是說不通。,魯迅(1881–1936)『書信集‧致孟十還』, publication date absent in Schmidt
```
Use en-dash instead of hyphen for range where relevant.

See _New Hart’s Rules_, 4.11.1.

@JaneXZTang